packages/opentelemetry-semantic-conventions/src/trace/SemanticAttributes.ts

```diff
@@ -2717,30 +3127,40 @@ const TMP_NETHOSTCONNECTIONTYPEVALUES_UNKNOWN = 'unknown';
 
 /**
  * The internet connection type currently being used by the host.
+ *
+ * @deprecated Use NET_HOST_CONNECTION_TYPE_VALUES_WIFI.
```

Comment:
@MSNev I suspect this was a mistake in the original PR that the whole name is a single uppercase block like `NETHOSTCONNECTIONTYPEVALUES_*`. Since we've released this it can't be removed so I just deprecated it in favor of the better names exported in `experimental.ts` and `stable.ts`

Reply:
No, it was a case of using the existing conversions to map the type names.

Reply:
And I don't think we should do this either as it will further conflate the issues for the build tools (ie the issue around the semantic conventions `client_id` and `client.id` being mapped to the same value), as this then cause the issue in that now we have CamelCase conversion could also cause name clashes

Reply:
I think it's actually (practically) impossible to clash? The attributes are prefixed with `ATTR_` which means in order to have a clash the attribute would have to have the word "values" in it _AND_ the enum would have to begin with the word "Attr". Each of these on their own seem quite unlikely, but together I think it's safe to say it will never happen.